src/platform/Linux/bluez/BluezEndpoint.h

```diff
@@ -43,39 +43,72 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-/**
- *    @file
- *          Provides Bluez dbus implementation for BLE
- */
-
 #pragma once
 
+#include <stdint.h>
+
+#include <gio/gio.h>
+#include <glib.h>
+
+#include <ble/CHIPBleServiceData.h>
 #include <lib/core/CHIPError.h>
-#include <system/SystemPacketBuffer.h>
 
+#include "BluezConnection.h"
 #include "Types.h"
 
 namespace chip {
 namespace DeviceLayer {
 namespace Internal {
 
+struct BLEAdvConfig;
+
+struct BluezEndpoint
+{
+    char * mpOwningName; // Bus owning name
+
+    // Adapter properties
+    char * mpAdapterName;
```

Comment:
since we already initialize some bits with nullpt, would it make sense to initialize all pointers in this struct to nullptr? and maybe even the integers to 0 and bools to false?